cypress/tests/ImageElement.spec.ts

```diff
@@ -41,13 +49,13 @@ describe("ImageElement", () => {
         getElementRichTextField("caption").should("have.text", text);
       });
 
-      it(`caption – should allow mark shortcuts in an element`, () => {
+      it.only(`caption – should allow mark shortcuts in an element`, () => {
```

Comment:
```suggestion
      it(`caption – should allow mark shortcuts in an element`, () => {
```

There's the culprit.

Reply:
Thanks for spotting 😅